src/components/EnvironmentOverride/ConfigMapOverrides.tsx

```diff
@@ -474,7 +474,7 @@ const OverrideConfigMapForm: React.FC<ConfigMapProps> = memo(function OverrideCo
                 subPath={subPath}
                 filePermission={filePermission}
             />}
-        {state.dialog && <ConfirmationDialog className="confirmation-dialog__body--w-360">
+        {state.dialog && <ConfirmationDialog className="confirmation-dialog__body--w-400">
```

Comment:
is this change intentional?

Reply:
Yes. Made the change as part of the UX suggestions.